### PR TITLE
Feature/fix anchor styling

### PR DESCRIPTION
**Pull request by sukhrajghuman**
### What this PR does

deleted sass code

### Checklist
- [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -7,42 +7,43 @@ import "./Footer.scss";
...
+						linkComponent={Link}
+						inline
+					/>
+				</AUfooterNav>
+				<AUfooterEnd>
+					© Commonwealth of Australia. With the exception of the
+					Commonwealth Coat of Arms and where otherwise noted, this
+					work is licensed under the{" "}
+					<a
+						href="https://raw.githubusercontent.com/adamzerella/ausgov/master/LICENSE"
```
Comment (azerella) [inline src/components/Footer.js]: We should change this to: `https://raw.githubusercontent.com/govau/ausgov/master/LICENSE`

Comment (azerella) [inline src/components/Footer.js]: The header doesn't seem to be centered anymore?

image

**Review by azerella — CHANGES_REQUESTED** [review]

File: src/components/Footer.js (hunk 1 above)
Comment (azerella) [reply src/components/Footer.js]:
```
const Header = ({ siteTitle }) => (
	<div className="header">
		<OfficalBanner />
		<AUheader>
			<div className="container-fluid">
...
```

Changing `<div className="container-fluid">`  to `<div className="container">` seems to fix the problem

**Comment by azerella** [comment]
Other than those changes looks good to go :)

File: src/components/Footer.js (hunk 1 above)
Comment (sukhrajghuman) [reply src/components/Footer.js]: yep because i deleted some sass code which was modifying the container fluid class.

Comment (sukhrajghuman) [reply src/components/Footer.js]: good pick up

**Pull request merged**